cmd/influxd/help.go

```diff
@@ -0,0 +1,35 @@
+package main
+
+import "fmt"
+
+// HelpCommand displays help for command-line sub-commands.
+type HelpCommand struct {
+}
+
+// NewHelpCommand returns a new instance of HelpCommand with default settings.
```

Comment:
The command doesn't have any settings, as far as I can see. This comment would probably be better if it just read "returns a new Instance of HelpCommand."